### PR TITLE
ADM-338 Skelton for Backend

### PR DESCRIPTION
**Pull request by huijunnnn**
## Before

the backend use TypeScript Node.js koa

## After

the backend use 
 java 
Spring boot
Testing(Junit 5)
1 health endpoint
swagger
logging framework(Log4j)
dockerfile

### CODE REVIEW

File: backend/src/main/java/heartbeat/controller/HealthController.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+@RestController
+@RequestMapping("/health")
+@RequiredArgsConstructor
+public class HealthController {
+    private final HealthEndpoint healthEndpoint;
+
+    @GetMapping
+    public Status getHealthStatus() {
+        return healthEndpoint.health().getStatus();
```
Comment (guzhongren) [inline backend/src/main/java/heartbeat/controller/HealthController.java:18]: Return a string or structure directly

File: backend/src/main/java/heartbeat/service/SimpleService.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package heartbeat.service;
+
+import lombok.extern.log4j.Log4j2;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.PostConstruct;
```
Comment (guzhongren) [inline backend/src/main/java/heartbeat/service/SimpleService.java:6]: Delete this file

**Review by guzhongren — CHANGES_REQUESTED** [review]

**Comment by guzhongren** [comment]
Test coverage?
PLZ give a screenshot for every task

File: backend/src/main/java/heartbeat/controller/HealthController.java (hunk 1 above)
Comment (Devonzhang) [reply backend/src/main/java/heartbeat/controller/HealthController.java:18]: Status is a object with a field code(serialized with name status), sample response for this endpoint is {"status": "UP"}

**Comment by Devonzhang** [comment]
@guzhongren test coverage
Screenshot 2023-01-11 at 12 33 25

**Review by guzhongren — APPROVED** [review]

File: backend/src/main/java/heartbeat/service/SimpleService.java (hunk 2 above)
Comment (Devonzhang) [reply backend/src/main/java/heartbeat/service/SimpleService.java]: Deleted unused service

**Pull request merged**